docs/explanation/archive.rst

```diff
@@ -0,0 +1,348 @@
+
+    <country-code>.archive.ubuntu.com
+
+You can see which mirror is the *Country Mirror* by doing a simple :term:`DNS` lookup. For example:
+
+.. tab-set::
+
+    .. tab-item:: Finland (FI)
+
+        .. code:: bash
+
+            dig fi.archive.ubuntu.com +noall +answer
+
+        .. code:: text
+
+            fi.archive.ubuntu.com.	332	IN	CNAME	mirrors.nic.funet.fi.
+            mirrors.nic.funet.fi.	332	IN	A	193.166.3.5
+
+        Therefore ``mirrors.nic.funet.fi`` is finlands *Country Mirror*.
+
+    .. tab-item:: Tunisia (TN)
+
+        Tunisia does not have any third-party mirrors in its country. Therefore the
+        Tunisia *Country Mirror* is just the main *Ubuntu Package Archive* server
+        (``archive.ubuntu.com``).
+
+        .. code:: bash
+
+            dig tn.archive.ubuntu.com +noall +answer
+
+        .. code:: text
+
+            tn.archive.ubuntu.com.	60	IN	A	185.125.190.36
+            tn.archive.ubuntu.com.	60	IN	A	91.189.91.83
+            tn.archive.ubuntu.com.	60	IN	A	91.189.91.82
+            tn.archive.ubuntu.com.	60	IN	A	185.125.190.39
+            tn.archive.ubuntu.com.	60	IN	A	91.189.91.81
+
+        which are just the ``archive.ubuntu.com`` IP addresses:
+
+        .. code:: bash
+
+            dig archive.ubuntu.com +noall +answer
+
+        .. code:: text
+
+            archive.ubuntu.com.	1	IN	A	185.125.190.39
+            archive.ubuntu.com.	1	IN	A	185.125.190.36
+            archive.ubuntu.com.	1	IN	A	91.189.91.83
+            archive.ubuntu.com.	1	IN	A	91.189.91.81
+            archive.ubuntu.com.	1	IN	A	91.189.91.82
+
+Package Uploads
+---------------
+
+:term:`Ubuntu` encourages contributions from any person in the wider community.
+However, direct uploading to the *Ubuntu Package Archive* is restricted. These
+general contributions need to be reviewed and uploaded by a :term:`Sponsor`.
+
+See the article :doc:`/explanation/sponsoring` that explains this process in more detail.
+
+Resources
+---------
+
+- `Ubuntu Release Cycle <https://ubuntu.com/about/release-cycle>`_
+- `Ubuntu Blog -- Ubuntu updates, releases and repositories explained <https://ubuntu.com/blog/ubuntu-updates-releases-and-repositories-explained>`_
+- `Ubuntu Server Docs -- Package Management <https://ubuntu.com/server/docs/package-management>`_
+- `Ubuntu Wiki -- Mirrors <https://wiki.ubuntu.com/Mirrors>`_
+
+- `Ubuntu Help -- Repositories <https://help.ubuntu.com/community/Repositories>`_
+- `Ubuntu Help -- Repositories/Ubuntu <https://help.ubuntu.com/community/Repositories/Ubuntu>`_
+
+Landscape Repositories
+~~~~~~~~~~~~~~~~~~~~~~
+
+Landscape is a management and administration tool for :term:`Ubuntu`.
+See more information `here <https://ubuntu.com/landscape>`_.
+
+Landscape allows you to mirror :term:`APT` :ref:`Repositories` like the
+*Ubuntu Package Archive*. Although it is not directly related to the
+*Ubuntu Package Archive* it can be educational to understand how
+:term:`APT` :ref:`Repositories` work in general.
```

Comment:
```suggestion
Landscape allows you to mirror APT repositories like the
Ubuntu package archive. Although it is not directly related to the
Ubuntu package archive, it can be educational to understand how
APT repositories work in general.
```